.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve OrganicMaps
```

Comment:
Describe your issue in details to help us improve Organic Maps

Reply:
Fixed